src/NsdPublicationChannelRegistryClient.js

```diff
@@ -30,6 +31,9 @@ const responseWithBody = (body, type, year) => {
 const performQuery = async (request, path, year) => {
   const type = path.substr(1, path.length)
   const nsdResponse = await axios.post(channelRegistryUri, request)
+  if (nsdResponse.status === 204) {
```

Comment:
const for 204?

Reply:
Using consts from http-status-codes in code an tests